projects/noloan/app/src/main/java/hasadna/noloan/SmsRecyclerAdapter.java

```diff
@@ -100,6 +105,13 @@ public void bind(SmsMessage sms) {
             intentToLawsuitForm.putExtra("body", sms.getBody());
             view.getContext().startActivity(intentToLawsuitForm);
           });
+
+      buttonAddSpam.setOnClickListener(
```

Comment:
Perhaps a better name would be `buttonAddSuggestion`  or `buttonSuggestSpam`?

Reply:
buttonSuggestSpam :+1: